### PR TITLE
update brand container storybook

### LINKED ISSUE #2194: Add container story for BrandContainer
**Is your feature request related to a problem? Please describe.**
As we have moved components over to the Psammead component library, we have few items in our Simorgh Storybook: https://bbc.github.io/simorgh/

We should include our container components in the Simorgh storybook

**Describe the solution you'd like**
Update story to BrandContainer (on storybook) - showing different context values e.g. News/Persian examples

- [x] What would be ideal, would be to add a drop-down that could switch between services and populate these with the correct view. e.g. selecting Persian from the drop-down when looking at the BrandContainer will show you the News | Persian brand.
- [x] The containers relating to RTL services, (e.g. Persian and in future, other RTL services) should have the `dir="rtl"` attribute passed through to Storybook. Could be added via a Decorator, that gets the value of `dir` from the Service Context. 

See the implementation of this for the Psammead sitewide links component - https://bbc.github.io/psammead/?path=/story/components-sitewidelinks--default When you select `Persian` from the `language` dropdown, it loads links in the right language and adds the correct `dir` value too.

Brand story Container: https://github.com/bbc/simorgh/blob/latest/src/app/containers/Brand/index.stories.jsx

**Describe alternatives you've considered**
N/A

**Testing notes**
Regression of the BrandContainer

Dev insight: Cypress tests may need to be updated, if so pair with test

**Additional context**
We shouldn't need stories for all of our containers (e.g. Blocks, or Fragments) since these are for handling our Optimo data.

### PR DESCRIPTION
**Pull request by thekp**
Resolves #2194

**Overall change:** _Update brand storybook to use knobs and have RTL for required services_

**Code changes:**

- _use storybook-helper to use inputProvider for brand storybook_
- _imported knobs_
- _imported storybook-helper_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/Brand/index.stories.jsx [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+storiesOf('Brand', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
+      null,
+      ({ service }) => {
+        return (
+          <ServiceContextProvider service={service}>
+            <Brand
```
Comment (dr3) [inline src/app/containers/Brand/index.stories.jsx]: Not sure this is right, the brand container only appears to take 2 props as far as I can tell https://github.com/bbc/simorgh/blob/latest/src/app/containers/Brand/index.jsx#L30-L33 so in storybook the new brand appears further right than the others

Comment (thekp) [reply src/app/containers/Brand/index.stories.jsx]: nice spot, i was just mindlessly updating the previous implementation of it. 
mindless

**Review by ryanmccombe — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by PriyaKR** [comment]
The hover state seems to be missing on the brand svg.

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**